setup.py

```diff
@@ -79,4 +78,7 @@
     install_requires=requirements,
     include_package_data=True,
     python_requires=">=3.5",
+    extras_require={
+        'neural-network': ["torch; sys_platform != 'win32'"]
```

Comment:
1. Neural networks are supported by the VQC algorithm, and this DOES NOT require torch
2. Neural networks are also supported by the qGAN algorithm, and this currently requires torch
So I am afraid that by using the `neural-network` flag people may think that — if they do not accept this optional package -- they will lose all the support for neural networks, which is not the case because they would still be able to use VQC.  Therefore, I suggest that we use a more specific flag here.
```suggestion
        'torch': ["torch; sys_platform != 'win32'"]
```